react/README.md

```diff
@@ -436,7 +436,7 @@ We don’t recommend using indexes for keys if the order of items may change.
       isPublished: false
     }
 
-    return (<div {...props} />);
+    return <div {...props} />;
```

Comment:
There are two examples where we use parents. I decided to remove them for consistency.

Reply:
Consistency is good when one form is preferred; in this case i think the inconsistency is better because it demonstrates that either form is acceptable